pcsx2/GameList.cpp

```diff
@@ -591,7 +591,7 @@ void GameList::ScanDirectory(const char* path, bool recursive, bool only_cache,
 	Console.WriteLn("Scanning %s%s", path, recursive ? " (recursively)" : "");
 
 	progress->PushState();
-	progress->SetFormattedStatusText("Scanning directory '%s'%s...", path, recursive ? " (recursively)" : "");
+	progress->SetFormattedStatusText(fmt::format(TRANSLATE_FS("GameList","Scanning directory {} {}..."), path, recursive ? TRANSLATE_STR("GameList","(recursively)") : "").c_str());
```

Comment:
This should probably be split into two strings to avoid the conditional `(recursively)` which could be problematic in some languages.